addons/account/models/account_move.py

```diff
@@ -2268,7 +2282,7 @@ def _onchange_name_warning(self):
     @api.onchange('journal_id')
     def _onchange_journal_id(self):
         if not self.quick_edit_mode:
-            self.name = '/'
+            self.name = '/' if self._get_last_sequence() or self.posted_before else False
```

Comment:
What is the difference between `'/'` and `False`?

Since we can have placeholders everywhere does it still make sense to have `/`  in the database? I believe it was only a technical solution to avoid having a empty cell, which was weird for accountants.
But removing this tech artifact could simplify quite a few things here and there.

Reply:
As we wanted to keep the behavior in the form view displaying "Draft" as the name when there is already a move in the sequence, i use False for the field to be empty and able to display a placeholder and '/' as the value that is detected in the template as the sign that "Draft" should be shown instead of the empty field with the placeholder (so that the name cannot be modified once the sequence is started).

Would you rather have '/' be set as the placeholder value to be detected instead of the name value ? and so never have the name '/' anymore ?